--- a/src/meson.build
+++ b/src/meson.build
@@ -7,7 +7,7 @@ gtk3_dir_dark   = join_paths(theme_dir_dark, 'gtk-3.0')
 gtk4_dir        = join_paths(theme_dir, 'gtk-4.0')
 gtk4_dir_dark   = join_paths(theme_dir_dark, 'gtk-4.0')

-sass = find_program('sass')
+sass = find_program('sassc')

 # Files can't have the same names in the build dir, so I need to put the meson junk in 2 dirs
 subdir('theme-light')
`